mode/examples/Topics/Geometry/Icosahedra/Icosahedra.pyde

```diff
@@ -23,13 +19,12 @@ icoY3Rot = PI / 200
 
 
 def setup():
+    global halfWidth, halfHeight, ico3XOffset, ico1XOffset, ico1, ico2, ico3
     size(640, 360, P3D)
-    global halfWidth, halfHeight, ico3XOffset, ico1XOffset
```

Comment:
I meant to have them all immediately following `size()`. My feeling is that declarations of symbols should *never* be hard to find (looks at C, shudders). Having globals, in particular, declared as soon as possible in a method, and grouped all together, makes them that much easier to see.